SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1893,8 +1893,8 @@ bool USpatialNetDriver::FindAndDispatchStartupOps(const TArray<Worker_OpList*>&
 		Worker_Op* AuthorityChangedOp = nullptr;
 		FindFirstOpOfTypeForComponent(InOpLists, WORKER_OP_TYPE_AUTHORITY_CHANGE, SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID, &AuthorityChangedOp);
 
-		// If we are going to get both ops, we expect them in the same Worker_OpList
-		check(AddComponentOp != nullptr || (AuthorityChangedOp == nullptr));
```

Comment:
Is there a reason this check wasn't firing in the contrived cube example?

Reply:
the AddComponentOp part of the check evaluates to true.  Unfortunately this check only covered the case where the authority op was received in an OpList by itself.